src/MainPage/MainPage.css

```diff
@@ -100,6 +100,33 @@ button:hover{
   transition: all 0.1s ease-in-out;
 }
 
+<<<<<<< HEAD
+.pokemon-details {
```

Comment:
What a cool design! definitely enhances the nostalgia feel. Thanks for getting this up and running before production.